asset/tx.go

```diff
@@ -59,16 +59,16 @@ func VirtualTxWithInput(virtualTx *wire.MsgTx, input *Asset,
 
 	txCopy := virtualTx.Copy()
 	txCopy.LockTime = uint32(input.LockTime)
-	txCopy.TxIn[zeroIndex].PreviousOutPoint.Index = idx
-	txCopy.TxIn[zeroIndex].Sequence = uint32(input.RelativeLockTime)
-	txCopy.TxIn[zeroIndex].Witness = witness
+	txCopy.TxIn[idx].PreviousOutPoint.Index = idx
```

Comment:
So if we're not doing the 1-in-1-out version, then we don't need to swap out information for each input. Instead, the virtual tx has all the inputs, and `idx` here is just passed to sign/verify.